src/static/css/_pagination.css

```diff
@@ -0,0 +1,55 @@
+.pagination {
+    background: var(--fl-bgColor, var(--pagination-bg));
+    border-block-start: 0.2rem solid var(--fl-fgColor, var(--pagination-border-color));
+    color: var(--fl-fgColor, var(--pagination-color));
+    display: flex;
+    font-family: var(--font-sans);
+    font-weight: 800;
+    justify-content: space-between;
+    padding: var(--space-m);
+    width: 20rem;
+}
+
+.pagination svg {
+    height: 2rem;
+    width: 2rem;
+}
+
+.pagination-previous svg {
+    transform: rotate(180deg);
+}
+
+.pagination-previous:hover {
+    background:
+        linear-gradient(
+            269.79deg,
+            var(
+                --fl-bgColor,
+                var(--pagination-bg)
+            ) -0.44%,
+            rgb(236 238 255 / 0%) 99.82%
+        );
+}
+
+.pagination-next:hover {
+    background: linear-gradient(90deg, var(--fl-bgColor, var(--pagination-bg)) 0%, rgb(236 238 255 / 0%) 100%);
```

Comment:
Same as above, I think the `rgb` value should be `var(--fl-bgColor, var(--color-white))`.